utils/docker/Dockerfile.el.9

```diff
@@ -65,7 +65,9 @@ RUN echo "no_proxy = $no_proxy"
 
 # Install latest versions of python tools.
 COPY requirements-build.txt requirements-utest.txt ./
-RUN python3 -m pip -vvv --no-cache-dir install --upgrade pip && \
+RUN if [ -n "$REPO_FILE_URL" ]; then direct="${REPO_FILE_URL##*//}; "\
+        direct="${direct%%/*}"; export no_proxy="${direct}"; fi; \
+    python3 -m pip -vvv --no-cache-dir install --upgrade pip && \
```

Comment:
Won't the `no_proxy` carry forward for remaining commands in this file? If so, I think it's better to decouple it from this RUN